contracts/mocks/ConvexYieldWrapperMock.sol

```diff
@@ -206,11 +206,9 @@ contract ConvexYieldWrapperMock is ERC20, AccessControl {
         uint256 collateral;
         Balances memory balance;
         for (uint256 i = 0; i < userVault.length; i++) {
-            if (userVault[i] != bytes12(0)) {
```

Comment:
if we revert the array cleanup in removeVault then we should add this back

Reply:
I like the cleanup